.github/workflows/tag.yml

```diff
@@ -46,7 +46,27 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: ${{ steps.docker_meta_img_hub.outputs.tags }}
-          labels: ${{ steps.docker_meta_img.outputs.labels }}
+          labels: ${{ steps.docker_meta_img_hub.outputs.labels }}
+
+      - name: Docker meta Service Name for docker hub (file storage)
+        id: docker_meta_img_hub_file_storage
+        uses: docker/metadata-action@v4
+        with:
+          images: docker.io/schulcloud/schulcloud-server, quay.io/schulcloudverbund/schulcloud-server
+          tags: |
+            type=semver,pattern={{version}},prefix=file-storage-,onlatest=false
+            type=semver,pattern={{major}}.{{minor}},prefix=file-storage-,onlatest=false
```

Comment:
Please over write the label `org.opencontainers.image.title` with an title who hat file-storage in it.
https://github.com/docker/metadata-action#overwrite-labels
https://github.com/hpi-schul-cloud/schulcloud-server/actions/runs/5784695322/job/15675891320#step:7:70